src/offline/cable_mpimaster_stub.F90

```diff
@@ -0,0 +1,32 @@
+!==============================================================================
+! This source code is part of the
+! Australian Community Atmosphere Biosphere Land Exchange (CABLE) model.
+! This work is licensed under the CSIRO Open Source Software License
+! Agreement (variation of the BSD / MIT License).
+!
+! You may not use this file except in compliance with this License.
+! A copy of the License (CSIRO_BSD_MIT_License_v2.0_CABLE.txt) is located
+! in each directory containing CABLE code.
+!
+! ==============================================================================
+! Purpose: Stub for the master driver when MPI is not available.
+!
+! ==============================================================================
+!
+MODULE cable_mpimaster
+  IMPLICIT NONE
+
+  PRIVATE
+  PUBLIC :: mpidrv_master
+
+CONTAINS
+
+  SUBROUTINE mpidrv_master(comm)
```

Comment:
Please add a docstring for the stub:
```suggestion
  SUBROUTINE mpidrv_master(comm)
    !! <docstring>
```
Preview: https://cable--469.org.readthedocs.build/en/469/api/proc/mpidrv_master~2.html

Reply:
Done.